Filter/Basic.php

```diff
@@ -203,13 +203,13 @@ public function getParameters()
 
     /**
      * Get the stream resource being filtered.
-     * Maybe available only during filter calls when the closing parameter is
-     * set to false.
+     * Maybe available only during `static::filter` calls when the closing
+     * parameter is set to false.
      *
      * @return  resource
      */
     public function getStream()
     {
-        return $this->stream;
+        return isset($this->stream) ? $this->stream : null;
```

Comment:
I don't really know how to use this class but this method should only be called during a particular phase and in a special context.
So I put an `isset` to avoid calls to invalid property.
My question was; Do you know if the line is it still correct with my fix?

The method is used by [`Hoa\Stream\Filter\LateComputed`#L96](https://github.com/hoaproject/Stream/blob/011ab91d942f1d7096deade4c8a10fe57d51c5b3/Filter/LateComputed.php#L96)
The php manual page: http://php.net/manual/en/php-user-filter.oncreate.php

I'll move that fix on the **doc & typo** commit instead of the **feature** commit, to be clearer.